### DISCUSSION TITLE
Bug report: --name and --adminName not working properly

### DISCUSSION THREAD

**Discussion by reorx** [vinceanalytics/vince#27] [General]
I tried to use a different admin username than "acme", but I got the following error

```
./vince admin --name admin --password something
./vince serve
…
2024/12/06 22:52:41 ERROR checking admin err="admin account not found"
```

then I deleted the data and tried to pass admin credentials to `serve` command directly:
```
./vince serve --adminName admin --adminPassword something
```

it started, but after logging in, I found the username is still "acme".

seems to be a simply problem of not handling the arguments well in the cli

**Comment by gernest** [comment]
Thanks for the report. I just fixed this and released anew patch for vince https://github.com/vinceanalytics/vince/releases/tag/v1.11.3

If you have time, can you update vince and confirm the issue is fully resolved?
